src/components/Header/Header.js

```diff
@@ -81,9 +83,10 @@ export const Header = () => {
                                             placeholder="Search podcasts..."
                                             onChange={filter}
                                         />
-                                        <span className="fa fa-sort-amount-asc sort btn"
```

Comment:
As concerned with this it was being touched too in the PR #44

Reply:
Ok @yokwejuste 
The changes I did is not much oh . Just changed but one line of codes.